src/main/java/jalau/at18/architects/CardComparatorForResource.java

```diff
@@ -0,0 +1,9 @@
+package jalau.at18.architects;
+
+public class CardComparatorForResource implements Comparator {
+    public boolean compare(Card c1, Card c2) {
```

Comment:
try using a diferent name for c1 and c2 because its not discriptive. you could use **cardOne** and **cardTwo**